pilot/pkg/security/authn/v1alpha1/policy_applier.go

```diff
@@ -117,7 +117,6 @@ func outputLocationForJwtIssuer(issuer string) string {
 }
 
 func convertToEnvoyJwtConfig(policyJwts []*authn_v1alpha1.Jwt) *envoy_jwt.JwtAuthentication {
-	var requirements []*envoy_jwt.JwtRequirement
 	providers := map[string]*envoy_jwt.JwtProvider{}
```

Comment:
is providers still being used ?

Reply:
yes, we don't need to specify the provider name when `AllowMissingOrFailed` is used.